File: WebApp/WebApp/Models/Line.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class Line
    {
        [Key]
        public string Id { get; set; }
        public string Direction { get; set; }
        public ICollection<Bus> Buses { get; set; }
        public ICollection<BusStopsOnLine> BusStopsOnLines { get; set; }
        public ICollection<PointPathLine> PointLinePaths { get; set; }
    }
}

File: angular-app/src/models/line.ts
import { Bus } from './bus';
import { BusStopsOnLine } from './bus-stops-on-line';
import { PointPathLine } from './point-path-line';

export class Line {
    public Id: string;
    public Direction: string;
    public Buses: Array<Bus> = new Array();
    public BusStopsOnLines: Array<BusStopsOnLine> = new Array();
    public PointLinePaths: Array<PointPathLine> = new Array();
    public DisplayName: string;
}

